nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/QueryDatabaseTable.java

```diff
@@ -437,7 +439,7 @@ public void processRow(ResultSet resultSet) throws IOException {
                 if (nrOfColumns > 0) {
                     for (int i = 1; i <= nrOfColumns; i++) {
                         String colName = meta.getColumnName(i).toLowerCase();
-                        String fullyQualifiedMaxValueKey = getStateKey(meta.getTableName(i), colName);
+                        String fullyQualifiedMaxValueKey = getStateKey(tableName, colName);
```

Comment:
Ugh, their driver doesn't include table name info in the result set?! I'll give this a try on MySQL, Oracle, and Postgres as well, to make sure there are no issues with case sensitivity, etc.  Good catch thanks!